contracts/Schema.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.6.0 <0.9.0;
+/**
+ * @title Schema
+ * @dev Schema contract
+ */
+contract SchemaRegistry {
+
+    struct Schema {
+        address issuer;
```

Comment:
Probably better naming will be an owner or creator instead of the issuer? As it is an account that created the schema

Reply:
the issuer of schema is an account that created this schema. It's ok

Reply:
Creator/owner makes here more sense than issuer for me too.

Reply:
fixed